### DISCUSSION TITLE
Remote display

### DISCUSSION THREAD

**Discussion by nonflammable** [tbnobody/OpenDTU#1697] [Ideas]
I think that with a small amount of work, it is possible to prepare and compile code for a remote display that retrieves current data from MQTT or Web API.
My OpenDTU is in the attic. I made a temporary set with ESP8266 and OLED display for my own needs, which shows current data from MQTT. The code is written in MicroPython and the way of presenting data differs significantly from that on OpenDTU.
Maybe it will be possible to prepare the OpenDTU code in such a way that it can be configured as a remote display?

**Comment by StefanOberhumer** [comment]
Do you mean something like https://github.com/tbnobody/OpenDTU/discussions/1685 ?

**Reply by nonflammable** [reply]
I don't mean the display in the browser, but a physical wireless display away from the OpenDTU motherboard. OLED connected to ESP8266 which gets data from MQTT or WebAPI.

**Reply by Tiese** [reply]
So, like [this](https://github.com/helgeerbe/OpenDTU-OnBattery/discussions/526)?!
